src/router_core/agent.c

```diff
@@ -39,27 +39,40 @@ static void qdr_manage_update_CT(qdr_core_t *core, qdr_action_t *action, bool di
 ALLOC_DECLARE(qdr_query_t);
 ALLOC_DEFINE(qdr_query_t);
 
+
+struct qdr_agent_t {
```

Comment:
since the object is already called qdr_agent_t is it redundant to have the word agent for its members ? For example, agent->agent_response_handler can just be called agent->response_handler, same with agent_subscription_mobile, agent_subscription_mobile, agent_subscription_local and agent_log_source

Reply:
I'm all for shorter names - I'll update the patch.